src/main/java/com/surveyMonkey/controllers/AdminController.java

```diff
@@ -135,4 +155,21 @@ public String deleteSurvey(@RequestParam("surveyCode") String surveyCode, @Reque
         }
         return "deleteSurvey";
     }
+    @PostMapping({"/deleteSurveyWithoutPswrd"})
```

Comment:
What is the purpose of delete survey without a password feature? Shouldn't be an admin/authorized only capability ? Shouldnt only the owner be allowed to delete the survey ?

Reply:
> What is the purpose of delete survey without a password feature? Shouldn't be an admin/authorized only capability ? Shouldnt only the owner be allowed to delete the survey ?

You are right only the owner should be able to delete survey, I didn't think of that when I was writing the code, good catch, thank you!